modules/web-console/docker/standalone/Dockerfile

```diff
@@ -63,22 +45,22 @@ WORKDIR /opt/web-console
 COPY build .
 
 # Install node modules.
-RUN cd /opt/web-console/frontend && npm install --no-optional --prod && npm run build
-RUN cd /opt/web-console/backend && npm install --only=production --no-optional
+RUN cd /opt/web-console/frontend && \
+    npm install --no-optional --prod && \
+    npm run build
 
-# Returns to base path.
-WORKDIR /opt/web-console
+RUN cd /opt/web-console/backend && \
+    npm install --only=production --no-optional
 
 # Copy nginx config.
-COPY ./nginx/nginx.conf /etc/nginx/nginx.conf
-COPY ./nginx/web-console.conf /etc/nginx/web-console.conf
+COPY nginx/* /etc/nginx/
 
-# Setup entrypoint.
-COPY ./entrypoint.sh .
-RUN chmod 755 /opt/web-console/entrypoint.sh
```

Comment:
Is this correct to remove this line if permissions on host machine is incorrect?

Reply:
As I see under Windows OS, executable flag exists, so there should be no problems.
```MINGW64 /c/Personal/code/incubator-ignite/modules/web-console/docker/standalone (ignite-7821)
$ git ls-files -s entrypoint.sh
100755 3882f06a2ed5e91006e3f94a3b1cca749b0ec10a 0       entrypoint.sh```